src/lib/components/forms/form/form-components/slider/pipes/tick-interval.pipe.ts

```diff
@@ -0,0 +1,14 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+/** @description
+ * A pipe, that returns how often to show ticks. It's relative to the step so that a tick always appears on a step.
```

Comment:
Was genau soll "relative to" hier aussagen?

Reply:
Ein tick bezieht sich zum Schritt.. also auf einem Step erscheint immer ein Tick bzw. Häkchen

Reply:
Okay, dann würde ich schreiben " It's related to...". Denn "relative" in diesem Sinne gibt es leider nicht.

Reply:
Alles klar 👍

Reply:
done